docs/streams/upgrade-guide.html

```diff
@@ -1735,73 +1735,38 @@ <h3 class="anchor-heading"><a id="streams_api_changes_0101" class="anchor-link">
 
     <h3 class="anchor-heading"><a id="streams_api_broker_compat" class="anchor-link"></a><a href="#streams_api_broker_compat">Streams API broker compatibility</a></h3>
 
-    <p>The following table shows which versions of the Kafka Streams API are compatible with various Kafka broker versions.</p>
+    <p>The following table shows which versions of the Kafka Streams API are compatible with various Kafka broker versions. For Kafka Stream version older than 2.3.x, please check <a href="/39/documentation/streams/upgrade-guide">3.9 upgrade document</a>.</p>
 
     <table border="1" class="non-scrolling-table docutils">
         <thead>
           <tr>
             <th></th>
-            <th colspan="5">Kafka Broker (columns)</th>
+            <th colspan="2">Kafka Broker (columns)</th>
           </tr>
         </thead>
         <tbody>
           <tr>
             <td>Kafka Streams API (rows)</td>
-            <td>0.10.0.x</td>
-            <td>0.10.1.x and 0.10.2.x</td>
-            <td>0.11.0.x and<br>1.0.x and<br>1.1.x and<br>2.0.x</td>
             <td>2.1.x and<br>2.2.x and<br>2.3.x and<br>2.4.x and<br>2.5.x and<br>2.6.x and<br>2.7.x and<br>2.8.x and<br>3.0.x and<br>3.1.x and<br>3.2.x and<br>3.3.x and<br>3.4.x and<br>3.5.x and<br>3.6.x and<br>3.7.x and<br>3.8.x and<br>3.9.x</td>
             <td>4.0.x</td>
           </tr>
           <tr>
-            <td>0.10.0.x</td>
+            <td>2.4.x and<br>2.5.x</td>
+            <td>compatible with exactly-once v2 turned off</td>
```

Comment:
2.4.x and 2.5.x do not have EOSv2 -- so this is confusing... There is nothing to be turned on/off -- EOSv2 was only added in 2.6.0 release.